engine/baml-lib/jsonish/src/deserializer/coercer/coerce_optional.rs

```diff
@@ -32,7 +32,7 @@ pub(super) fn coerce_optional(
         Some(v) => match inner.coerce(ctx, optional_target, Some(v)) {
             Ok(v) => Ok(v),
             Err(e) => {
-                flags.add_flag(Flag::DefaultButHadUnparseableValue(e));
+                flags.add_flag(Flag::DefaultButHadUnparsableValue(e));
```

Comment:
```suggestion
                flags.add_flag(Flag::DefaultButHadUnparseableValue(e));
```

both "parseable" and "parsable" are valid spellings, but the former will show up when grepping/searching for "parse"